pom.xml

```diff
@@ -43,11 +43,13 @@
             <groupId>net.kyori</groupId>
             <artifactId>adventure-text-minimessage</artifactId>
             <version>4.0.0-SNAPSHOT</version>
+            <scope>compile</scope>

```

Comment:
Scopes werden hier absichtlich ausgelassen, weil im Dist-Modul alles "compile" ist.

Reply:
Außerdem: Gibt es einen besonderen Grund, wieso Du einen alten Snapshot benutzt? Laut dem Badge in der Repo ist 4.3.0 die neuste und stabile Version.

Reply:
4.1.0 ist (laut dem Badge auf Github und dem Repo) die neueste Version, außer für den Bungee-Serializer, da ist es 4.0.0. Aber meine war trotzdem veraltet, also passt :D

Reply:
Jap, ich habe es mit der falschen Repo verwechselt. Trotzdem ist es schade, dass eine stabile Version scheinbar keine Priorität hat. Es wäre schön gewesen, wenn ein Release reproduzierbar bliebe. Aber okay